src/AppInstallerCLICore/Workflows/UninstallFlow.cpp

```diff
@@ -123,4 +123,70 @@ namespace AppInstaller::CLI::Workflow
 
         context.Reporter.Info() << Resource::String::UninstallFlowUninstallSuccess << std::endl;
     }
+
+    void ReportDependencies(Execution::Context& context)
+    {
+        context <<
+            Workflow::GetManifest <<
```

Comment:
`GetManifest` is not the right thing to do here.  I still think that this should use the unified context data item, so this particular task wouldn't worry about it.

But when you do get the dependencies, it should be something like this:
```
auto p = context.Get<Execution::Data::Package>();
if (matchingAvailableVersion = p->GetAvailableVersion({"", p->GetInstalledVersion()->GetProperty(Version), "" }))
{
  ChooseInstallerBasedOnInstalledVersion(matchingAvailableVersion->GetManifest()).Dependencies;
}
```